horovod/common/timeline.h

```diff
@@ -50,15 +50,19 @@ class TimelineWriter {
   void Initialize(std::string file_name,
                   std::chrono::steady_clock::time_point start_time_);
   void Shutdown();
-  inline bool IsHealthy() const { return healthy_; }
-  inline bool Active() const { return active_; }
   void EnqueueWriteEvent(const std::string& tensor_name, char phase,
                          const std::string& op_name, const std::string& args,
                          long ts_micros);
   void EnqueueWriteMarker(const std::string& name, long ts_micros);
   void SetPendingTimelineFile(std::string filename);
+  short active();
```

Comment:
Why `short` instead of `bool`?

Reply:
Some c++ gimmick. Weirdly fetch or applies to atomic integral types. "This member function is only defined in the atomic specializations for integral types (except for bool)." from  http://www.cplusplus.com/reference/atomic/atomic/fetch_or/